inc/composer/class-command.php

```diff
@@ -172,6 +172,8 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
 			return $this->shell( $input, $output );
 		} elseif ( $subcommand === 'import-uploads' ) {
 			return $this->import_uploads( $input, $output );
+		} elseif ( $subcommand === 'cloud-build' ) {
```

Comment:
Suggestion: `check-build`?

Reply:
Why not just `build`?

Reply:
I think the intention is to test that a build runs without errors, and doesn't actually produce a build that can be deployed to Altis, so `build` may be a bit misleading.